### PR TITLE
NMS-12599: Added documentation on PDF dashboards

### PR DESCRIPTION
**Pull request by Bonrob2**
Documentation on how to create a PDF dashboard. Also made some edits to the existing file. NMS-12599

### All Contributors

* [ ] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [ ] Have you made a comment in that issue which points back to this PR?
* [ ] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [ ] If this is a new or updated feature, is there documentation for the new behavior?
* [ ] If this is new code, are there unit and/or integration tests?
* [ ] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

### Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

Once the reviewer(s) accept the PR and the branch passes continuous integration in Bamboo, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/${JIRA-ISSUE-NUMBER}
* Bamboo (Continuous Integration): https://bamboo.opennms.org/

### CODE REVIEW

File: opennms-doc/guide-admin/src/asciidoc/text/database-reports/database.adoc [hunk 1]
```diff
@@ -264,17 +264,58 @@ To get the appropriate `interface descriptor` depends on the input parameter.
...
-         A SSL secured connection can be established even if `org.opennms.netmgt.jasper.measurement.ssl.enable` is set to `false`.
+         An SSL secured connection can be established even if `org.opennms.netmgt.jasper.measurement.ssl.enable` is set to `false`.
 
 [[ga-database-reports-limitations]]
 === Limitations
 
  * Only a _JDBC Datasource_ to the _{opennms-product-name} Database connection_ can be passed to a report, or no datasource at all.
    One does not have to use the datasource, though.
+
+=== Creating PDF Reports from Grafana Dashboards Using OpenNMS
```
Comment (indigo423) [inline opennms-doc/guide-admin/src/asciidoc/text/database-reports/database.adoc:282]: > ... Using "OpenNMS"
Is it OpenNMS Horizon, Meridian should we use the {opennms-product-name} variable here? This here is not a blocker and also not very critical. I've just noticed while reading.

File: opennms-doc/guide-admin/src/asciidoc/text/database-reports/database.adoc [hunk 2]
```diff
@@ -264,17 +264,58 @@ To get the appropriate `interface descriptor` depends on the input parameter.
...
+
+OpenNMS provides three templates to create a PDF report from an existing Grafana dashboard. You can also schedule and email these PDF reports to anyone:
```
Comment (indigo423) [inline opennms-doc/guide-admin/src/asciidoc/text/database-reports/database.adoc:284]: one-line per sentence

File: opennms-doc/guide-admin/src/asciidoc/text/database-reports/database.adoc [hunk 3]
```diff
@@ -264,17 +264,58 @@ To get the appropriate `interface descriptor` depends on the input parameter.
...
+
+* Keep staff without access to OpenNMS informed about network performance for improved capacity planning
```
Comment (indigo423) [inline opennms-doc/guide-admin/src/asciidoc/text/database-reports/database.adoc:286]: "OpenNMS" same here, usage of the {opennms-product-name} for OpenNMS Horizon / OpenNMS Meridian.

File: opennms-doc/guide-admin/src/asciidoc/text/database-reports/database.adoc [hunk 4]
```diff
@@ -264,17 +264,58 @@ To get the appropriate `interface descriptor` depends on the input parameter.
...
+* Create a permanent record of strategic information and progress over a long period of time
+
+The PDF report displays each of the panels from the specified dashboard, with one, two, or four panels per page, depending on the selected template. 
+
+NOTE: This feature requires OpenNMS and an instance of Grafana with at least one dashboard and panel. OpenNMS allows you to create a report for any Grafana dashboard, not just those created using link:http://docs.opennms.org/helm/branches/master/helm/latest/welcome/index.html[OpenNMS Helm].
```
Comment (indigo423) [inline opennms-doc/guide-admin/src/asciidoc/text/database-reports/database.adoc:291]: "OpenNMS" same here, usage of the {opennms-product-name} for OpenNMS Horizon / OpenNMS Meridian.

File: opennms-doc/guide-admin/src/asciidoc/text/database-reports/database.adoc [hunk 5]
```diff
@@ -264,17 +264,58 @@ To get the appropriate `interface descriptor` depends on the input parameter.
...
+
+To create a PDF from a dashboard, follow these steps:
+
+. In the OpenNMS UI, choose `Reports>Database Reports`.
```
Comment (indigo423) [inline opennms-doc/guide-admin/src/asciidoc/text/database-reports/database.adoc:295]: "OpenNMS" same here, usage of the {opennms-product-name} for OpenNMS Horizon / OpenNMS Meridian.

File: opennms-doc/guide-admin/src/asciidoc/text/database-reports/database.adoc [hunk 6]
```diff
@@ -264,17 +264,58 @@ To get the appropriate `interface descriptor` depends on the input parameter.
...
+. In the `Report Templates` area, click `Grafana Dashboard Report <Xppp>`, where `<Xppp>` represents the number of panels per page you want to display. 
+. In the `Report Parameters` area, specify the appropriate information.
+.. Note that `Grafana Endpoint` is the datasource. Select a Grafana dashboard from the drop-down list. 
+.. You can also specify CSV for report type. 
+. Click `Create Report`. 
+.. You are prompted to save the report locally or open it.
+The file is saved to a folder on the OpenNMS Server [what folder?].
```
Comment (indigo423) [inline opennms-doc/guide-admin/src/asciidoc/text/database-reports/database.adoc:302]: "OpenNMS" same here, usage of the {opennms-product-name} for OpenNMS Horizon / OpenNMS Meridian.

File: opennms-doc/guide-admin/src/asciidoc/text/database-reports/database.adoc (hunk 2 above)
Comment (Bonrob2) [reply opennms-doc/guide-admin/src/asciidoc/text/database-reports/database.adoc:284]: ARGH! Thanks. I try to remember. :-)

**Comment by Bonrob2** [comment]
Documentation on creating PDF dashboard reports.

**Comment by Bonrob2** [comment]
Documentation on creating PDF dashboard reports.

**Pull request merged**